package.json

```diff
@@ -9,7 +9,9 @@
   "scripts": {
     "start": "npm run build && node dist/index.js",
     "build": "rm -rf dist && babel server/ --out-dir dist/ --ignore ./node_modules",
-    "start:dev": "nodemon --exec babel-node server/index.js"
+    "start:dev": "nodemon --exec babel-node server/index.js",
+    "test": "set NODE_ENV=test && jest --coverage --forceExit",
```

Comment:
Why `forceExit`. I think `--exit` flag will stop it

Reply:
I will try it.